insights/parsers/lvm.py

```diff
@@ -301,6 +301,8 @@ class PvsHeadings(LvmHeadings):
     PRIMARY_KEY = Pvs.PRIMARY_KEY
 
     def parse_content(self, content):
+        self.warnings = set(find_warnings(content))
```

Comment:
@akshay196 please add this new attribute to the docstring Attributes section.

Reply:
Done